File: tweet.py
class Tweet:

    def __init__(self):
        self.id = None
        self.content = None
        self.timestamp = None
        self.user = None

    def __repr__(self):
        return '<Tweet:user=%s, content=%s, id=%s, timestamp=%s>' % (
            self.user, self.content, self.id, self.timestamp
        )

File: src/ontology/src/models/Tweet.java
package models;

public class Tweet {

	String id;
	String username;
	//String name;
	String content;
	String timestamp;

	public String getId() {
		return id;
	}
	public void setId(String id) {
		this.id = id;
	}
	public String getUsername() {
		return username;
	}
	public void setUsername(String username) {
		this.username = username;
	}
	/*
	public String getName() {
		return name;
	}
	public void setName(String name) {
		this.name = name;
	}*/
	public String getContent() {
		return content;
	}
	public void setContent(String content) {
		this.content = content;
	}
	public String getTimestamp() {
		return timestamp;
	}
	public void setTimestamp(String timestamp) {
		this.timestamp = timestamp;
	}
	
}
